src/ui/MobileMenu/MobileEmojisBottomSheet.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { ReactElement, useMemo, useState } from 'react';
+import { EmojiContainer } from '@sendbird/chat';
+import { Member } from '@sendbird/chat/groupChannel';
+import { FileMessage, Reaction, UserMessage } from '@sendbird/chat/message';
+
+import './mobile-menu-emojis.scss';
+
+import BottomSheet from '../BottomSheet';
+import { getEmojiUrl } from '../../utils';
+import ImageRenderer from '../ImageRenderer';
+import Icon, { IconColors, IconTypes } from '../Icon';
+import Label, { LabelColors, LabelTypography } from '../Label';
+import { useChannelContext } from '../../modules/Channel/context/ChannelProvider';
+import UserListItem from '../UserListItem';
+
+export interface MobileEmojisBottomSheetProps {
+  message: UserMessage | FileMessage;
+  emojiKey: string;
+  hideMenu: () => void;
+  emojiContainer?: EmojiContainer;
+}
+
+export const MobileEmojisBottomSheet = (props: MobileEmojisBottomSheetProps): ReactElement => {
+  const {
+    message,
+    emojiKey = '',
+    hideMenu,
+    emojiContainer,
+  } = props;
+  const { currentGroupChannel } = useChannelContext();
+  const { members = [] } = currentGroupChannel;
+  const [selectedEmoji, setSelectedEmoji] = useState(emojiKey);
+
+  const ReactorList = useMemo(() => {
+    const memberList = message.reactions?.find(reaction => reaction.key === selectedEmoji)
+      ?.userIds.map((userId) => members.find((member) => member.userId === userId) ?? null)
+      .filter((member) => member !== null) as Array<Member>;
```

Comment:
`as Array<Member> ` is this type-casting really necessary?

Reply:
It says the memberList type is `Array<Member | null>` even I filtered null